### PR TITLE
Add model A of VANS equations

### PR DESCRIPTION
**Pull request by tonielgeitani**
This PR adds model A of the VANS equations to the solver. It also add a max local mass source post processing term.

### CODE REVIEW

File: source/fem-dem/parameters_cfd_dem.cc
Comment (shahabgol) [inline source/fem-dem/parameters_cfd_dem.cc:120]: Did modelB show higher accuracy?

File: source/fem-dem/parameters_cfd_dem.cc
Comment (shahabgol) [inline source/fem-dem/parameters_cfd_dem.cc:158]:
```suggestion
      throw(std::runtime_error("Invalid vans model. Valid choices are modelA and modelB."));
```

**Review by shahabgol — COMMENTED** [review]
Very clean PR. I have two minor comments. Why the default vans model is modelB? I was expecting modelA to be the default model.

File: source/fem-dem/parameters_cfd_dem.cc
Comment (tonielgeitani) [reply source/fem-dem/parameters_cfd_dem.cc:120]: No, both show similar behavior. I just put it as default but we can change it. If we put model A as default than all vans and cfd-dem tests will fail because the output will change slightly. That is why I didn't do that.

**Review by shahabgol — APPROVED** [review]
Well done!

File: source/fem-dem/parameters_cfd_dem.cc
Comment (blaisb) [reply source/fem-dem/parameters_cfd_dem.cc:120]: We already did the development initially for model B, so I think that's a correct choice to keep B as default. To be honest, we no clue yet which one might be better...

File: include/fem-dem/parameters_cfd_dem.h
Comment (blaisb) [inline include/fem-dem/parameters_cfd_dem.h:93]:
```suggestion
    VANSModel    vans_model;
```

File: include/fem-dem/parameters_cfd_dem.h
Comment (blaisb) [inline include/fem-dem/parameters_cfd_dem.h:56]:
```suggestion
  enum class VANSModel
```

**Review by blaisb — COMMENTED** [review]
Very few comments. Very good work. Once the grad-div PR is merged we can merge this one (once all test pass) :)

**Pull request merged**
